converter/src/main/java/gov/cms/qpp/conversion/decode/QppXmlDecoder.java

```diff
@@ -59,15 +59,13 @@ public QppXmlDecoder(Context context) {
 	 */
 	@Override
 	public DecodeResult decode(Element element, Node parentNode) {
-		if (null == element) {
+		if (element == null) {
 			return DecodeResult.ERROR;
 		}
 
 		setNamespace(element, this);
 
-		DecodeResult decodeResult = decodeChildren(element, parentNode);
-
-		return (decodeResult != null) ? decodeResult : DecodeResult.TREE_CONTINUE;
+		return decodeChildren(element, parentNode);
```

Comment:
We are losing the possibility of not returning `DecodeResult.TREE_CONTINUE` if `decodeChildren` returns `null`.

Reply:
All tests pass, so I'm assuming this should never happen & is therefore a pointless check. I can readd it if you guys want, but I don't see much point.